src/Components/OATPropertyEditor/FormRootModelDetails.tsx

```diff
@@ -220,6 +231,51 @@ export const FormUpdateProperty = ({
         setIsAMultiLanguageDescriptionEmpty(hasEmptyValues);
     }, [multiLanguageSelectionsDescription]);
 
+    const onIdInputFocus = () => {
+        if (idInputRef.current) {
+            const selectionStart =
+                idInputRef.current.props.value.lastIndexOf(':') + 1;
+            const selectionEnd = idInputRef.current.props.value.lastIndexOf(
+                ';'
+            );
```

Comment:
I've added a validation in case the index return -1.
In case of the semicolon before the colon, the dtmi regex should trigger. Let me know if this is ok